config/autoload/zend-expressive.global.php

```diff
@@ -3,9 +3,6 @@
 return [
     'debug' => false,
 
-    // Cache the configuration. Recommended for production.
-    'config_cache_enabled' => false,
```

Comment:
Considering the issues we had on #zftalk when ZF3 was released, I prefer to disable caching by default. There were a lot of questions and issues where the config was cached (enabled by default) and changes that were made were not applied to the app since the cache was loaded.

I try to prevent that by having the cache disabled by default and they need to read what it does to enable it.